### ISSUE TITLE
Adding adapter package to Bot Framework Composer (experimental version) removes other installed adapter

### ISSUE THREAD

**Issue by Clooney24** [microsoft/BotFramework-Composer#7589]
## Describe the bug

Adding adapter package to Bot Framework Composer (experimental version) removes other installed adapter. I want to add Twilio, Slack and Facebook, but installing the second removes the first installed adapter

## Version

Version:  2.0.0-nightly.242275.686a144
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## OS

Windows

## To Reproduce

1. Install Twilio adapter package
2. configure it
3. Install Facebook or Slack package
4. See that Twilio is no longer installed

## Expected behavior

Install multiple adapter packages

**Comment by srinaath** [comment]
@carlosscastro Could you clarify the behavior here?

**Comment by Clooney24** [comment]
When I first install Twilio adapter and then install Slack adapter, I expect both adapters to be installed as both channels could be used simultanously. But after installing the second adapter, the first is no longer installed. I mean it's not listed in "Packages"->"Installed" anymore and it is not listed in "Project Configuration"->"Connections" any more. Looks like second installation replaces the first one.

(as I just checked, in "Packages"->"Installed" even that one installed last, that is still listed in "Connections" will not be listed as installed package. so this might be 2 problems)

**Comment by carlosscastro** [comment]
Hello @Clooney24 both adapters should work side by side. Could you share the 'runtimeSettings' section of your appsettings.json (removing any sensitive data)?

In the meantime, I'll try your scenario on my side as well.

**Comment by Clooney24** [comment]
Sorry, I already downgraded to 1.4.1 as we have a short deadline for this project.

**Comment by carlosscastro** [comment]
No problem, we'll investigate shortly and update here.

**Comment by carlosscastro** [comment]
Hello @Clooney24 I just did the following steps:

1. Upgraded composer to v2.0.0-nightly.243234.8264ac4 (latest version, just clicked 'Check for updates' in the composer app)
2. Create a new blank bot
3. Install Twillio adapter
4. Install Slack adapter
5. Verified that both appear in the connections pane and I can configure both
![image](https://user-images.githubusercontent.com/21318528/117345300-3e92d800-ae5b-11eb-9905-a6bffc9aa34f.png)

From your other related bug, I suspect that you were upgrading an old project? If yes, that may explain the strange behavior. On new bots it should just work as I just tried. If you do want to get your migrated bot project to use these adapters I'm happy to assist you in the future, just let me know here -- a good starting point is sharing the 'runtimeSettings' of your appsettings.json if you want to go that route. Another option is to create a new bot to ensure you have the latest template and then import your assets.
